code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -305,17 +305,18 @@
 	new /obj/item/handcuffs(src)
 	update_icon()
 
+
 /obj/item/storage/belt/holster/security/fullusp45/New()
 	..()
 	new /obj/item/reagent_containers/spray/pepper(src)
 	new /obj/item/device/flash(src)
 	new /obj/item/device/flashlight/maglight(src)
-	new /obj/item/gun/projectile/pistol/usp45(src)
 	new /obj/item/ammo_magazine/scp/usp45(src)
 	new /obj/item/ammo_magazine/scp/usp45(src)
 	new /obj/item/ammo_magazine/box/acp45(src)
 	new /obj/item/handcuffs(src)
 	update_icon()
+	new /obj/item/gun/projectile/pistol/usp45(src)
```

Comment:
Why move this to the end?
I'd put update_icon() after it, just to be safe

Reply:
Tried to fix an issue where the overlay didn't apply properly
Regardless it had the same issue
I'll put it back as it was